support/ebpf/types.h

```diff
@@ -328,6 +334,7 @@ typedef enum TracePrograms {
   PROG_UNWIND_RUBY,
   PROG_UNWIND_V8,
   PROG_UNWIND_DOTNET,
+  PROG_GO_LABELS,
```

Comment:
Maybe just a personal nit, but if we add more progs for labels, it would be nice, if could use something like this:
```suggestion
  PROG_LABELS_GO,
```

Reply:
Yeah we can wait and see where this goes when the native labels PR goes up, I don't have strong feelings either way, we might just drop the "GO" bit.